industrial_ci/src/docker.sh

```diff
@@ -163,7 +163,7 @@ function ici_prepare_docker_image() {
 #   (None)
 
 function ici_build_default_docker_image() {
-  export DOCKER_IMAGE="industrial-ci/$OS_CODE_NAME"
+  export DOCKER_IMAGE="industrial-ci/${OS_CODE_NAME:-${DOCKER_BASE_IMAGE##*:}}"
```

Comment:
Perhaps this change is not necessary.
However, `DOCKER_BASE_IMAGE` has to be provided anyway!